.github/workflows/release.yml

```diff
@@ -73,3 +73,37 @@ jobs:
         if: steps.changesets.outputs.published != 'true'
         # TODO: this command will be replaced by the changesets snapshots feature
         run: yarn lerna publish 0.0.0-canary-${{ github.sha }} --exact --dist-tag canary --yes --no-push --force-publish "*"
+
+  docker:
+    runs-on: ubuntu-latest
+
+    if: contains(jobs.npm.steps.changesets.outputs.publishedPackages.*.name, '@commercetools-frontend/mc-http-server')
```

Comment:
I hope we can reference across jobs. Haven't seen many do that but my understanding is that it should work :)